### PR TITLE
feat(connect): add option for multiple account select

### PR DESCRIPTION
**Pull request by abyigiter**

### CODE REVIEW

**Review by mucahit — CHANGES_REQUESTED** [review]
@yigiterdev Thanks! I have a couple of comments

1. Let's rename the flag to `shouldSelectSingleAccount:boolean = false`

```diff
- allowToSelectMultipleAccounts
+ shouldSelectSingleAccount
```

2. The same flag also should be sent to the Web Wallet so that we can have the same functionality on both applications. _The way you send the flag to Mobile Wallet is seems correct by the way._

3. Let's have an optional options parameter for the `connect` method to reduce possible future breaking changes over there. 

```diff
- connect(allowToSelectMultipleAccounts?: boolean)
+ connect({shouldSelectSingleAccount}: PeraWalletConnectMethodOptions = {})
```

4. And finally let's document the new option in the related Readme section.

File: src/modal/peraWalletConnectModalUtils.ts [hunk 1]
```diff
@@ -41,11 +42,28 @@ function createModalWrapperOnDOM(modalId: string) {
...
+    close: () => removeModalWrapperFromDOM(PERA_WALLET_CONNECT_MODAL_ID)
+  };
+}
+
 function openPeraWalletConnectModal(modalConfig: PeraWalletModalConfig) {
   return (uri: string) => {
     if (!document.getElementById(PERA_WALLET_CONNECT_MODAL_ID)) {
       const root = createModalWrapperOnDOM(PERA_WALLET_CONNECT_MODAL_ID);
-      const newURI = `${uri}&algorand=true`;
+      const newURI = `${uri}&algorand=true&allowToSelectMultipleAccounts=${modalConfig.shouldSelectSingleAccount}`;
```
Comment (mucahit) [inline src/modal/peraWalletConnectModalUtils.ts:66]:
```suggestion
      const newURI = `${uri}&algorand=true&shouldSelectSingleAccount=${modalConfig.shouldSelectSingleAccount}`;
```

**Review by mucahit — COMMENTED** [review]
Thanks!

**Comment by mucahit** [comment]
Waiting for the Readme change to merge this @yigiterdev

**Comment by yigiterdev** [comment]
Readme updated. @mucahit

**Review by mucahit — APPROVED** [review]

**Pull request merged**